lib/google/amp/cache/version.rb

```diff
@@ -1,7 +1,9 @@
+# frozen_string_literal: true
+
 module Google
   module AMP
     module Cache
-      VERSION = "0.1.0"
+      VERSION = '0.1.0'
```

Comment:
Should we release a new version for these changes?

Reply:
Yes, for sure. Gonna cut a new version later.